src/pages/html_conversions/new_page.cr

```diff
@@ -35,13 +35,29 @@ class HtmlConversions::NewPage < PageLayout
     end
   end
 
+  private def render_clear_button
```

Comment:
Added a simple Clear button next to Copy, so you don't need to either delete the content of the textarea or reload the page